initial-promises/node-server/common/1.0/site.cf

```diff
@@ -83,6 +83,9 @@ bundle common g
                                               "curl/libcurl.dll",
                                               "curl/libssl32.dll",
                                               "curl/libeay32.dll",
+                                              "openssl/openssl.exe",
+                                              "openssl/libeay32.dll",
```

Comment:
Having an up to date openssl library for both should be a good thing so i think you should put the most recent library here.
And try not to duplicate binaries since they already are pretty heavy